adapters/yieldmo/yieldmo.go

```diff
@@ -136,3 +136,13 @@ func NewYieldmoBidder(endpoint string) *YieldmoAdapter {
 		endpoint: endpoint,
 	}
 }
+
+func getMediaTypeForImp(impId string, imps []openrtb.Imp) openrtb_ext.BidType {
+	//default to banner unless video exists in impression
+	for _, imp := range imps {
```

Comment:
Does the Yeildmo bid response include the bid type? Although the code you have here is common in the project, there's been issues reported (see https://github.com/prebid/prebid-server/issues/862) that this approach isn't always reliable for multiformat impressions. Is there enough information to detect the type based on the bid response?

Reply:
This is a good point. We unfortunately do not currently have anything in our responses that would be a good signal for determining the media type.

Our bidder doesn't support multi format requests at this time. I don't see any evidence on our end that we are receiving requests with multi format impressions, but if it were to happen, you are correct that the implementation I used would've resulted in that bug. In the event that we received both, we would've only considered the banner request, but the code in the adapter would've marked it as video, because video was present in the request.

I have flipped the logic in the adapter, so that banner is always used if banner is present. This mirrors the behavior on our end, and will prevent us from incorrectly marking bids. I recognize that in the long term, this is still a fragile way to doing this, so I have added a ticket on our end to include the media type in an extension object of the bid response. Once we have gotten that through our release process, I can make another update to our adapter to use that field instead. Will this work, for now?

Reply:
Perfect. Thank you.